WooCommerce/Classes/ViewRelated/Survey/SurveyViewController.swift

```diff
@@ -72,6 +72,7 @@ extension SurveyViewController {
                 return WooConstants.URLs.inAppFeedback
                     .asURL()
                     .tagPlatform("ios")
+                    .tagAppVersion(Bundle.main.bundleVersion())
```

Comment:
Wasn't sure if we should be using `bundleVersion` or `shortVersionString`... lmk! 😄